boards/nucleo32-f031/include/periph_conf.h

```diff
@@ -174,21 +174,6 @@ static const spi_conf_t spi_config[] = {
 #define RTC_NUMOF           (0U)
 /** @} */
 
-/**
- * @name   ADC configuration
- * @{
- */
-#define ADC_CONFIG {            \
```

Comment:
I guess @aabadie means, currently there are 5 ADC lines defined here, but in your common definition in `periph_conf_nucleo32.h` above you define 8 ADC lines.